packages/fxa-content-server/app/scripts/templates/sign_up_password.mustache

```diff
@@ -35,16 +35,16 @@
       {{{ coppaHTML }}}
 
       {{^isCWTSOnSignupPasswordEnabled}}
-      <section class="get-involved">
+      <section class="mb-5">
       {{#isAnyNewsletterEnabled}}
           <header>
-              <h2>{{#t}}Practical knowledge is coming to your inbox. Sign up for more:{{/t}}</h2>
+              <h2 class="mb-5 text-base ltr:text-left rtl:text-right">{{#t}}Practical knowledge is coming to your inbox. Sign up for more:{{/t}}</h2>
```

Comment:
For my knowledge: how do we decide when to add `ltr:` and `rtl:`?

Reply:
@chenba I think generally speaking if anything is left-aligned it's expected to be right-aligned in RTL languages so it all reads the same, I usually just try to picture flipping the entire page in the other direction. We have a note about it [here](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/README.md#rtl-support). (it's probably time that readme was moved into ecosystem-platform...)